package.json

```diff
@@ -40,7 +40,7 @@
   },
   "devDependencies": {
     "bower": "1.5.2",
-    "enb": ">= 0.15.0 < 1.0.0",
+    "enb": ">= 1.2.0 < 2.0.0",
```

Comment:
Not really. Here we can use mock-i18n tech to write, let's say, `.mock-lang.js` file.